src/Libraries/Microsoft.Extensions.Http.Resilience/FaultInjection/Internal/HttpClientFaultInjectionMetrics.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Extensions.Http.Resilience.FaultInjection.Internal;
+
+// TODO: can be also a generic Http[Client]ResilienceMetrics class
```

Comment:
@martintmk do you have any thoughts on a better name here?

Reply:
I think current name is fine.